front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -199,6 +201,12 @@ const AdminProjectEventEdit = () => {
     }
   }, [remoteEventFiles]);
 
+  const containerRef = useCallback((node) => {
+    if (node !== null) {
+      setWidth(node.getBoundingClientRect().width);
+    }
+  }, []);
+
```

Comment:
This logic seems to be used a couple of times, does it need to be a shared function?